config/jobs/kubernetes/sig-cloud-provider/gcp/sig-gcp-windows.yaml

```diff
@@ -135,6 +135,42 @@ periodics:
       - --timeout=120m
       image: gcr.io/k8s-testimages/kubekins-e2e:v20190906-fc85258-master
 
+  annotations:
+    testgrid-dashboards: google-windows, sig-windows
+    testgrid-tab-name: gce-windows-1.16
+    description: Runs tests on a Kubernetes 1.16 cluster with Windows nodes on GCE"
+- name: ci-kubernetes-e2e-windows-gce-k8sstable1
```

Comment:
My thinking is let 1.16 point to ci/k8s-beta, 1.15 & 1.14 point to ci/k8s-stable1 and they have the same test group name. Not sure if it's ok.

Reply:
Yeah, it looks like the blocks from lines 142-176 and 178-212 are the same except for the annotations part right now. Looks like line 198 needs to change to `--extract=ci/k8s-stable2`, and you can also change the name at line 178 to match.

@yujuhong do you know where the meaning of --extract is explained, or where "stable1" and "stable2" are defined? @YangLu1031 you might find some information in the runner.sh or kubernetes_e2e.py scripts to help explain what these flags are doing.

Reply:
Found in kubernetes_e2e.py, looks like it's the GCS bucket name storing the k8s binaries.
```
# tell kubetest to extract from this 
        shared_build_gcs_path = read_gcs_path(gcs_path)
                args.kubetest_args.append('--extract=' + shared_build_gcs_path)
                args.build = None
                break
```

Reply:
@yujuhong @pjh Thanks, made the changes, how can we see the results on testgrid for 1.16?

Reply:
Right. I don't know if there's an easy way to preview the test job / testgrid changes before merging.